Block/Js.php

```diff
@@ -94,6 +132,16 @@ protected function getJsLibLocation()
         return $this->helper->getJsLibLocation();
     }
 
+    /**
+     * Get Sentry Enabled
+     *
+     * @return mixed
```

Comment:
shouldn't this always return a boolean?

Reply:
oops forgot to change those when copy/pasting, good catch thanks!